### PR TITLE
[12.0][MIG] stock_picking_operation_quick_change

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
It fails when clicking on "Change Location" button. Error below:

`KeyError: 'stock.picking.operation.wizard'`

**Comment by MiquelRForgeFlow** [comment]
I don't have this error. Please, check again.

File: stock_picking_operation_quick_change/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Stock Picking Operation Quick Change",
+    "summary": "Change location of all picking operations",
+    "version": "12.0.1.0.0",
+    "category": "Inventory",
```
Comment (rousseldenis) [inline stock_picking_operation_quick_change/__manifest__.py]: Change this to 'Warehouse' to get it in good category in odoo-community.org/shop

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Minor comment. Tested on runbot

File: stock_picking_operation_quick_change/__manifest__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply stock_picking_operation_quick_change/__manifest__.py]: done

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: It works, my previous review was wrong.

**Review by rousseldenis — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**